README.md

```diff
@@ -6,11 +6,79 @@
 ## Summary
 This protected microservice provides a backend for Making Tax Digital for Businesses (MTDfB) VAT frontend services to retrieve VAT Return Obligation details for a user enrolled for MTD VAT.
 
+## Endpoint Definitions (APIs)
+
+**Method**: GET
+
+**Path**: /enterprise/obligation-data/:vatNumber
```

Comment:
We want the path to be the route in this microservice, rather than the DES route.